spec/publish_a_course_spec.rb

```diff
@@ -111,11 +111,11 @@
     expect(page.find(".govuk-heading-xl ")).to have_content(course_name)
 
     course_code = course_name.gsub("Primary with geography and history (", "").gsub(")", "")
-    find_course_regex = /https:\/\/qa.find-postgraduate-teacher-training.service.gov.uk\/course\/\S*\/#{course_code}/
+    find_course_regex = /#{ENV.fetch("FIND_URL")}\/course\/\S*\/#{course_code}/
```

Comment:
It'd be good to use [Regexp.escape](https://docs.ruby-lang.org/en/master/Regexp.html#method-c-escape) here to make sure we don't inadvertently parse this string as a regex rule.

```suggestion
    find_course_regex = /#{Regexp.escape ENV.fetch("FIND_URL")}\/course\/\S*\/#{course_code}/
```